apps/docs/app/ui/main-nav.tsx

```diff
@@ -72,15 +60,26 @@ const mainNavItems = [
   },
 ];
 
-export const MainNav = () => (
-  <nav
-    className="-order-1 w-72 max-w-full bg-sky-lightest px-5 py-9"
-    aria-label="Navigasjonsmeny for grunnmuren"
-  >
-    <ul>
-      {mainNavItems.map((mainNavItem) => (
-        <MainNavItem key={mainNavItem.title} {...mainNavItem} />
-      ))}
-    </ul>
-  </nav>
-);
+export const MainNav = () => {
+  const routeApi = getRouteApi('/_docs');
+  const { data } = routeApi.useLoaderData();
```

Comment:
Dette lar oss aksessere data returnerert fra en loaderen i layouten. [Se tanstack router docs](https://tanstack.com/router/v1/docs/framework/react/guide/data-loading#consuming-data-from-loaders).

For å være mer eksplisitt burde kanskje dataene sendes fra layouten til denne komponenten, men jeg syntes det var en kul hook som jeg ville teste 😅